README.rst

```diff
@@ -199,6 +199,28 @@ For details of the JDBC driver options refer to the official documentation.
 
 NOTE: Option names and values are case-sensitive. The option value is specified as a character string.
 
+Specify the Query Results
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to specify where athena stores the `txt` and `txt.metadata` files containing the result of each query, you can specify it as follows:
+
+.. code:: python
+
+    from pyathenajdbc import connect
+
+    conn = connect(s3_staging_dir='s3://YOUR_S3_BUCKET/path/to/',
+                   region_name='us-west-2',
+                   LogPath='/path/to/pyathenajdbc/log/',
+                   LogLevel='6',
+                   S3OutputLocation='s3://YOUR_S3_BUCKET/path/to/query_results/' )
+
+For details of the JDBC driver options refer to the official documentation.
```

Comment:
The following link is not a document of the JDBC driver options.

Reply:
ah yes, sorry. that was copy paste from above.